libs/net/dal/Migrations/1.0.5/Up/PreUp/00-Tag.sql

```diff
@@ -0,0 +1,37 @@
+DO $$
+BEGIN
+
+CREATE TEMP TABLE _tag AS
+SELECT "id", "name"
+FROM public.tag;
+
+ALTER TABLE _tag
+ADD "new_id" integer;
+
+CREATE SEQUENCE seq START 1;
+UPDATE _tag
+SET new_id = nextval('seq');
+
+-- set id of publig.tag to match id of _tag table
+UPDATE public.tag
+SET "id" = _tag.new_id
+FROM _tag
+WHERE _tag.name = public.tag.name;
+
+-- update content_tag table to match new id
+UPDATE public.content_tag
+SET "tag_id" = _tag.new_id
+FROM _tag
+WHERE _tag.id = public.content_tag.tag_id;
+
+-- drop foreign key constraint on content_tag table
+ALTER TABLE public.content_tag DROP CONSTRAINT "FK_content_tag_tag_tag_id";
```

Comment:
Don't you need to to this before you make the above two update statements?

Reply:
Only threw an error when attempting to change data type before dropping constraint

Reply:
You have to remove the relationship before you change primary key values.  I don't know how this can work.